scripts/.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
we should be able to re-use the root-level eslint configuration instead of needing to manage another one

Reply:
Using the root `.eslintrc` config will throw 340 lint errors:

- Unexpected console statement
- Synchronous fs calls should not be used in Cypress

We could certainly ignore these other ways but I feel like a separate config for these scripts makes sense given these warnings. Thoughts?

Reply:
What changed that is is failing with this PR using the same eslint config file and not in develop?

Reply:
@jordanpowell88 I found this PR [here](https://github.com/cypress-io/cypress/pull/26197) from a few months ago that was adding linting to the scripts directory.  In it, a `no-console` of `off` was added to an overrides section in the main `.eslintrc.js` file.  Should this now be removed?